bot.js

```diff
@@ -53,11 +53,11 @@ client.on('messageReactionAdd', async (reaction, user) => {
 
         if (foundUser) {
             foundUser.increment('points', { by: 5 });
-            reaction.message.channel.send(`Congrats ${foundUser.username}, you just earned 5 points! (Total points: ${foundUser.points + 5})`);
+            reaction.message.channel.send(`Congrats <@${reaction.message.author.id}>, <@${user.id}> just awarded you 5 points! (Total points: ${foundUser.points + 5})`);
```

Comment:
Do we want this and line 60 to be a shared variable maybe since they're both sending the same message?

Reply:
Sure, we can do that.

Reply:
We could take the beginning portion of the message before the total points and set that to a shared variable. That would eliminate the redundancy.